src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameService.java

```diff
@@ -95,7 +98,12 @@ public void runTurn(Game currentGame){
         inMemoryGameRepository.save(currentGame);
     }
 
-
+    private void createCardCollection(Game currentGame){
+        User host = UserContextHolder.getCurrentUser();
+        CardCollection CardCollection = new CardCollection(currentGame.getGameParameters(), host.getSpotifyJWT().getAccessToken());
+        currentGame.setCardCollection(CardCollection);
+        inMemoryGameRepository.save(currentGame);
+    }
```

Comment:
Was handled in the constructor of Game previously. For me it seems fine to move it into GameService for more flexibility but then we should remove it in the constructor.